src/components/block-tunes/block-tune-move-down.ts

```diff
@@ -5,9 +5,9 @@
  */
 
 import { API, BlockTune } from '../../../types';
-import Popover from '../utils/popover';
 import { IconChevronDown } from '@codexteam/icons';
 import { TunesMenuConfig } from '../../../types/tools';
+import { PopoverItem } from '../utils/popover/popover-item';
```

Comment:
PopoverItem should be a private class I guess, so it should not be accessed from outside